crypto/fipsmodule/ec/p521.c

```diff
@@ -433,18 +429,6 @@ OPENSSL_STATIC_ASSERT(P521_MUL_WSIZE == 5,
 
 // p521_select_point_affine selects the |idx|-th affine point from
 // the given precomputed table and copies it to |out| in constant-time.
```

Comment:
Should this comment also be removed?

Reply:
yes, I removed it in the next PR: https://github.com/aws/aws-lc/pull/2004